crud/select.lua

```diff
@@ -86,14 +97,8 @@ local function select_iteration(space_name, conditions, opts)
     return results
 end
 
-local function get_replicasets_to_select_from(plan, all_replicasets)
-    if not plan.is_scan_by_full_sharding_key_eq then
-        return all_replicasets
-    end
-
-    plan.scanner.limit = 1
-
-    local bucket_id = vshard.router.bucket_id_strcrc32(plan.scanner.value)
+local function get_replicaset_by_scan_value(scan_value)
```

Comment:
It confuses me. It's get_replicasets_by_sharding_key.

To be `by_scan_value` this function should include logic that scan_value is full specified sharding key and returns routeall if it's not so. But it doesn't.

Reply:
Please, se the commit below. I've fixed full scan key case processing.